app/presenters/tree_node/physical_server.rb

```diff
@@ -0,0 +1,5 @@
+module TreeNode
+  class PhysicalServer < Node
+    set_attribute(:text, &:name)
```

Comment:
You don't need this line at all, it's inherited from the `Node` class automagically.